src/tools/miri/cargo-miri/src/util.rs

```diff
@@ -169,10 +169,8 @@ where
 }
 
 pub fn ask_to_run(mut cmd: Command, ask: bool, text: &str) {
-    // Disable interactive prompts in CI (GitHub Actions, Travis, AppVeyor, etc).
-    // Azure doesn't set `CI` though (nothing to see here, just Microsoft being Microsoft),
-    // so we also check their `TF_BUILD`.
-    let is_ci = env::var_os("CI").is_some() || env::var_os("TF_BUILD").is_some();
```

Comment:
I guess this will affect users who run Miri on Azure.

Reply:
Yeah. Azure CI still exists, right? If so, `cargo miri` should recognize it.